case-api-client/src/main/java/uk/gov/ons/ctp/integration/caseapiclient/caseservice/model/RmCaseDTO.java

```diff
@@ -1,13 +1,12 @@
 package uk.gov.ons.ctp.integration.caseapiclient.caseservice.model;
 
-import com.fasterxml.jackson.annotation.JsonFormat;
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-
 import java.time.LocalDateTime;
 import java.util.List;
 import java.util.Map;
 import java.util.UUID;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
```

Comment:
Is this needed? 
Just wondering if Eclipse now complain about an unused import

Reply:
But it is used?